server/service/src/sync/test/test_data/purchase_order.rs

```diff
@@ -176,8 +176,8 @@ fn purchase_order_1_push_record() -> TestSyncOutgoingRecord {
             communications_charge: None,
             insurance_charge: Some(1.0),
             freight_charge: None,
-            supplier_discount_amount: 20.0,
-            order_total_before_discount: 400.0,
+            supplier_discount_amount: 18.0,
```

Comment:
The test still expects 20.0 here to match the legacy record
```suggestion
            supplier_discount_amount: 20.0,
```